pillar/heroku/discussions.sls

```diff
@@ -85,6 +87,7 @@
       'CLOUDFRONT_DIST': 'd2mcnjhkvrfuy2',
       'CORS_URLS': ['https://micromasters.mit.edu', 'https://ocwnext.odl.mit.edu', 'https://ocw-beta.odl.mit.edu', 'https://ocw-preview.odl.mit.edu', 'https://ocw-published.odl.mit.edu', "https://draft.ocw.mit.edu", "https://www.ocw.mit.edu", "https://ocw.mit.edu", 'https://live.ocw.mit.edu'],
       'DEBUG': False,
+      'ENABLE_INFINITE_CORRIDOR': False,
```

Comment:
This sets the values in the Jinja dictionary, but you'll still need to use it to set the key in the YAML dict that gets used for populating the Heroku settings.